scout/utils/ensembl_biomart_clients.py

```diff
@@ -29,15 +29,6 @@ def stream_get(self, url: str) -> Iterator:
     def stream_resource(self, interval_type: str) -> Iterator[str]:
         """Use schug web to fetch genes, transcripts or exons from a remote Ensembl biomart in the right genome build and save them to file."""
 
-        def yield_resource_lines(iterable) -> str:
-            """Removes the last element from an iterator."""
-            it = iter(iterable)
-            current = next(it)
-            for i in it:
-                yield current
-                current = i
-
         shug_url: str = f"{SCHUG_BASE}{SCHUG_RESOURCE_URL[interval_type]}{self.build}"
 
-        # return all lines except the last, which contains the "[success]" string
-        return yield_resource_lines(self.stream_get(shug_url))
+        return self.stream_get(shug_url)
```

Comment:
Ahem, I feel very convinced I approved this first time around? But, I guess it was a busy day. 😊

Reply:
It's OK, I am not in a rush